### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/authentication#6]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)
## Results for https://pages-staging.18f.gov/authentication/:
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.63:1. Recommendation: change background to #317ab9.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div > form > button
  - `<button type="submit" class="btn">Sign in with a MyUSA Account</button>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.04:1. Recommendation: change text colour to #6d6d6d.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > footer
  - `<footer> Secured with <a href="https:/...</footer>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.04:1. Recommendation: change text colour to #6d6d6d.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > footer > a
  - `<a href="https://github.com/bitly/oauth2_proxy#oauth2_proxy">OAuth2 Proxy</a>`
## Summary:
- 3 Errors
